zitadel/local/README.md

```diff
@@ -0,0 +1,166 @@
+Fill all required fields and then click the button `Create` to create the new user.
+![create-user-for-xpanse](compose/statics/create-user-for-xpanse.png)
+
+> Note: The first login requires a password change, please remember the new password.
+
+Follow the steps below to grant all roles of 'xpanse' for the new admin user:
+1. Select item `Authorizations` under the user and then click on the button `+ New N` to start the grant;
+![grant-user-roles-start](compose/statics/grant-user-roles-start.png)
+2. Select the project `eclipse-xpanse`and then click on the button `Continue`;
+![img.png](compose/statics/grant-select-project.png)
+3. Select all roles to the grant and then clock on the button `Save`;
+![img.png](compose/statics/grant-select-all-roles.png)
+After completing the grant above, user has roles as below:
+![img.png](compose/statics/view-user-has-all-roles.png)
+
+## Run Applications of Xpanse with Local Service of Zitadel
+
+### Config and run the API service of Xpanse.
+
+Open the project of [Xpanse](https://github.com/eclipse-xpanse/xpanse.git) with the IDEA for local development.
+Create a new properties file named 'application-zitadel-local.properties' under the `resources` of the module `runtime`,
+then copy the context of Terraform outputs to fill in, and then clean all `"` and ` ` as below:
+![create-zitadel-local-properties](compose/statics/create-zitadel-local-properties.png)
+
+Update the value of config item 'spring.profiles.active' in the default 'application.properties', and then run the main 
+class 'XpanseApplication'.
+![run-xpanse-with-local-zitadel.png](compose/statics/run-xpanse-with-local-zitadel.png)
+
+Open internet browser and navigate to http://localhost:8080/swagger-ui/index.html. Verify that the Xpanse API is 
+properly connected to the local service of Zitadel by Authorization on the page of Swagger UI. The browser will redirect
+to the login page of local service of Zitadel with URL http://localhost:8081/ui/login/login. You can use the created
+admin user to login and call all APIs on the Swagger UI.
+![authorize-on-swagger-ui.png](compose/statics/authorize-on-swagger-ui.png)
+
+### Config and run the UI client of Xpanse.
+
+Open the project of [Xpanse-ui](https://github.com/eclipse-xpanse/xpanse-ui.git) with the IDEA for local development.
+Copy the config file '.env.zitadel-testbed' and save as a new env file named '.env.zitadel-local' under the root path
+of the project `xpanse-ui`, then update the context of config item as below:
+
+* The value of item 'REACT_APP_ZITADEL_AUTHORITY_NAME' copied from item 'authorization-server-endpoint' in Outputs;
+* The value of item 'REACT_APP_ZITADEL_CLIENT_ID' copied from item 'authorization-xpanse-ui-client-id' in Outputs;
+![copy-env-file-update-item](compose/statics/copy-env-file-update-item.png)
+
+Edit the config file 'package.json', copy the scripts 'start-with-zitadel-testbed' and 'build-with-zitadel-testbed', and
```

Comment:
we must remove this step.

by default the `npm start` command must use the values in .env file. So it is enough to update this file to point to port 8081

https://github.com/swaroopar/xpanse-ui/blob/5ecb08b8486caa2068f97770b78d80dad98fed1b/.env#L2

Reply:
Ok.